docs/snippets/_sdk-reduce-latency.mdx

```diff
@@ -1,7 +1,5 @@
-By default, UID2 SDKs make API calls to a UID2 production environment server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
+In this scenario, depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
```

Comment:
Not keen on talking about Singapore or global in this snippet, would rather than info was in the common Environments page

Reply:
@jon8787: updated. The detailed info is in the Environments page only, and other pages link to there. other than the implementation examples. Hope this is what you wanted. Snippet is unnecessary in this scenario, because of nuances of differences between pages.